core/designsystem/src/main/kotlin/com/dhc/designsystem/Color.kt

```diff
@@ -2,10 +2,42 @@ package com.dhc.designsystem
 
 import androidx.compose.ui.graphics.Color
 
-val Purple80 = Color(0xFFD0BCFF)
-val PurpleGrey80 = Color(0xFFCCC2DC)
-val Pink80 = Color(0xFFEFB8C8)
+data class SurfaceColor(
+    val neutral30: Color = Color(0xFFF4F4F5),
+    val neutral50: Color = Color(0xFFE6EDF8),
+    val neutral100: Color = Color(0xFFD7E1EE),
+    val neutral200: Color = Color(0xFFA5B2C5),
+    val neutral300: Color = Color(0xFF7B8696),
+    val neutral700: Color = Color(0xFF1F2127),
+    val neutral800: Color = Color(0xFF17191F),
+    val neutral900: Color = Color(0xFF0F1114)
+)
 
-val Purple40 = Color(0xFF6650a4)
-val PurpleGrey40 = Color(0xFF625b71)
-val Pink40 = Color(0xFF7D5260)
+data class AccentColor(
+    val violet50: Color = Color(0xFFE6E8FF),
+    val violet100: Color = Color(0xFFCDD1F2),
+    val violet200: Color = Color(0xFFB5BAEB),
+    val violet300: Color = Color(0xFF939BE2),
+    val violet400: Color = Color(0xFF5E69D4),
+    val violet500: Color = Color(0xFF5660C1),
+    val violet600: Color = Color(0xFF414BAE),
+    val violet700: Color = Color(0xFF343FA6)
+)
+
+data class TextColors(
+    val textMain: Color,
+    val textBodyPrimary: Color,
+    val textHighLightsSecondary: Color,
+    val textHighLightsPrimary: Color
+)
+data class BackgroundColors(
+    val backgroundMain: Color
+)
+
+
+data class DhcColors(
+    val surface: SurfaceColor,
+    val accent: AccentColor,
+    val text: TextColors,
+    val background: BackgroundColors
+)
```

Comment:
@ddyeon 
컬러들을 나눈 이유가 있나 ??
머터리얼 테마 컬러도 primary, onPrimary, background 등을 다른 클래스에 구분하지 않고 사용하는데, 비슷한 구조로 가면 더 좋을 것 같단 말이지 ??
지금 구조에서는 컬러를 찾을 때 1depth가 더 늘어나는 느낌 ...?

```suggestion
object DhcColor {
    val neutral30: Color = Color(0xFFF4F4F5)
    val neutral50: Color = Color(0xFFE6EDF8)
    val neutral100: Color = Color(0xFFD7E1EE)
    ...
    val violet50: Color = Color(0xFFE6E8FF)
    val violet100: Color = Color(0xFFCDD1F2)
    val violet200: Color = Color(0xFFB5BAEB)
    ...
}

object DhcThemeColor {
    val textMain = DhcColor.neutral30
    val textBodyPrimary = DhcColor.neutral100
    ...
}
```

개인적인 의견이니깐 컬러를 명확히 구분하기 위한 의도가 있었다면 넘어가도 좋습네다 ~

Reply:
전자는 다른 타입의 테마컬러가 추가되는 경우 유연하게 확장할 수 있고, 후자는 타입이 하나만있을 때 효과적일 수 있을수도 있겠넹

과연 다연이의 선택은?!

Reply:
저는 타입의 확장성을 고려해서 전자로 가겠습니다!!!!!! 나중에 백그라운드나 텍스트 컬러가 묶어놔야지 보기 좋을 것 같아..ㅎㅎ